server/services/interventionsService.test.ts

```diff
@@ -527,6 +525,40 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
       expect(referral.id).toBe('dfb64747-f658-40e0-a827-87b4b0bdcfed')
       expect(referral.additionalRiskInformation).toEqual('A danger to the elderly')
     })
+
+    it('returns the updated referral when setting the service category ID', async () => {
```

Comment:
Should we have one that tests we can set the first name, too?

Reply:
isn't there already a test for updating SU info?

Reply:
Nope - we're just able to receive it, not set it. I just wanted the property there on the model so that I could use it in presenters.

Reply:
(I've just been cheekily setting that property in the code when testing things)

Reply:
ah i see. for now i might just remove the SU bit from the initial data, and address in another PR after george has done the work on getting SU data from ndelius. then maybe the initial data is the CRN, not SU name.